File: AV1_AULA3/C++/contador.cpp
#include <bits/stdc++.h>
using namespace std;
  
void fun()
{
	for(int i = 1; i <= 1000; i++){
		cout << i << endl;
	}
}
  
int main(){
	

    time_t start, end;
    fun();
    
    double time_taken = double(end - start);
    cout << "O tempo de execução do programa foi de: " << fixed
         << time_taken << setprecision(5);
    cout << " seg " << endl;
    return 0;
}

File: AV1_AULA3/contador.py
#Faça um programa que imprima os números de um 1 até 1000 e ao final deve verificar o tempo que cada um leva para ser executado

import time
inicio=time.time()


for i in range(1001):    
    print(i)
fim = time.time()


diferenca = fim - inicio

print('O tempo de execução foi: ',diferenca)
